COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
Because according my terms of employment say that they own the copyright on anything I produce while I work for them.

If you question is why I'm including them in this file? Because that's part of the terms for me (=them) to contribute here.

What it means? It just naming one (out of many) other copyright holders but Eelco. (Unless there's a CLA that reassigns all copyright to him that I missed.)